extension/src/setup/studio.ts

```diff
@@ -205,30 +210,47 @@ export class Studio extends Disposable {
     })
   }
 
+  private resetStudioValues(
+    previousStudioUrl: string,
+    previousStudioAccessToken: string | undefined
+  ) {
+    this.studioAccessToken = undefined
+    this.shareLiveToStudio = undefined
+    this.studioUrl = DEFAULT_STUDIO_URL
+
+    if (previousStudioAccessToken || previousStudioUrl !== DEFAULT_STUDIO_URL) {
```

Comment:
[Q] what about `shareLiveToStudio`?

Reply:
Updated to check for `shareLiveToStudio`!